# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/forms/display-block-baselines-3-ref.html
# Commit: 2187935f6d78
# Full Hash: 2187935f6d78c22ceeb7e5e92e919aeb42820b95
# Author: David Shin <dshin@mozilla.com>
# Date: 2023-08-16 03:43:43
# Regressor Bug: 1846982
# File Overlap Count: 1
# Description:
#   Bug 1846982: Baseline calculation for textarea. r=emilio
#   
#   Tests are added as tentative, because the behaviour when there's no content,
#   or when there's placeholder text, seem very differnt for each browser.
#   
# ==============================================================================

diff -r 2d475d0406cd -r 2187935f6d78 layout/reftests/forms/display-block-baselines-3-ref.html
--- a/layout/reftests/forms/display-block-baselines-3-ref.html	Tue Aug 15 13:26:25 2023 +0000
+++ b/layout/reftests/forms/display-block-baselines-3-ref.html	Tue Aug 15 13:30:17 2023 +0000
@@ -32,12 +32,16 @@
   overflow: visible;
 }
 
+.grid {
+  display: inline-grid;
+  baseline-source: last;
+}
   </style>
 </head>
 <body>
 
 <div>
-  <div style="display:inline-grid">
+  <div class="grid">
     A
     <textarea class="block">textarea</textarea>
   </div>
@@ -45,7 +49,7 @@
 </div>
 
 <div>
-  <div style="display:inline-grid">
+  <div class="grid">
     A
     <textarea class="block no-theme">textarea</textarea>
   </div>
@@ -53,7 +57,7 @@
 </div>
 
 <div>
-  <div style="display:inline-grid">
+  <div class="grid">
     A
     <textarea class="block no-theme no-scroll">textarea</textarea>
   </div>
@@ -61,7 +65,7 @@
 </div>
 
 <div>
-  <div style="display:inline-block">
+  <div class="grid">
     A<br>
     <div style="display:inline-block"><fieldset style="display:inline">fieldset-first<br>fieldset-last</fieldset></div>
   </div>